package_manifest.yaml

```diff
@@ -140,115 +232,19 @@ foreman_packages:
-      - koji-foreman-nightly
-    nodejs-axios: {}
-    nodejs-babel-cli: {}
-    nodejs-babel-core: {}
-    nodejs-babel-loader: {}
-    nodejs-babel-plugin-lodash: {}
-    nodejs-babel-plugin-transform-class-properties: {}
-    nodejs-babel-plugin-transform-object-assign: {}
-    nodejs-babel-plugin-transform-object-rest-spread: {}
-    nodejs-babel-polyfill: {}
-    nodejs-babel-preset-env: {}
-    nodejs-babel-preset-react: {}
-    nodejs-babel-register: {}
-    nodejs-brace: {}
-    nodejs-c3: {}
-    nodejs-change-emitter: {}
-    nodejs-classnames: {}
-    nodejs-compression-webpack-plugin: {}
-    nodejs-css-loader: {}
-    nodejs-datatables.net: {}
-    nodejs-datatables.net-bs: {}
-    nodejs-debug: {}
-    nodejs-diff: {}
-    nodejs-dotenv: {}
-    nodejs-expose-loader: {}
-    nodejs-extract-text-webpack-plugin: {}
-    nodejs-fbjs: {}
-    nodejs-file-loader: {}
-    nodejs-follow-redirects: {}
-    nodejs-harmony-reflect: {}
-    nodejs-hoist-non-react-statics: {}
-    nodejs-identity-obj-proxy: {}
-    nodejs-ipaddr.js: {}
-    nodejs-is-buffer: {}
-    nodejs-isomorphic-fetch: {}
-    nodejs-jquery: {}
-    nodejs-jquery-flot: {}
-    nodejs-jquery-ujs: {}
-    nodejs-jquery.cookie: {}
-    nodejs-js-tokens: {}
-    nodejs-jsdom: {}
-    nodejs-jstz: {}
-    nodejs-lodash: {}
-    nodejs-lodash-webpack-plugin: {}
-    nodejs-loose-envify: {}
-    nodejs-ms: {}
-    nodejs-multiselect: {}
-    nodejs-node-gyp: {}
-    nodejs-node-sass: {}
-    nodejs-object-assign: {}
-    nodejs-patternfly: {}
-    nodejs-patternfly-react: {}
-    nodejs-performance-now: {}
-    nodejs-prop-types: {}
-    nodejs-raf: {}
-    nodejs-react: {}
-    nodejs-react-bootstrap: {}
-    nodejs-react-bootstrap-switch: {}
-    nodejs-react-c3js: {}
-    nodejs-react-dom: {}
-    nodejs-react-ellipsis-with-tooltip: {}
-    nodejs-react-fontawesome: {}
-    nodejs-react-numeric-input: {}
-    nodejs-react-onclickoutside: {}
-    nodejs-react-password-strength: {}
-    nodejs-react-redux: {}
-    nodejs-recompose: {}
-    nodejs-redux: {}
-    nodejs-redux-form: {}
-    nodejs-redux-form-validators: {}
-    nodejs-redux-logger: {}
-    nodejs-redux-thunk: {}
-    nodejs-sass-loader: {}
-    nodejs-seamless-immutable: {}
-    nodejs-select2: {}
-    nodejs-style-loader: {}
-    nodejs-symbol-observable: {}
-    nodejs-urijs: {}
-    nodejs-url-loader: {}
-    nodejs-uuid: {}
-    nodejs-webpack: {}
-    nodejs-webpack-stats-plugin: {}
-    nodejs-zxcvbn: {}
-    puppet-agent-oauth: {}
-    puppet-agent-puppet-strings: {}
-    puppet-agent-yard: {}
-    rubygem-foreman_api: {}
-    rubygem-foreman_maintain: {}
-    rubygem-kafo: {}
-    rubygem-kafo_parsers: {}
-    rubygem-kafo_wizards: {}
-    rubygem-rb-inotify: {}
-    rubygem-rsec: {}
-    rubygem-rubyipmi: {}
-katello_packages:
   vars:
-    package_base_dir: 'packages/katello/'
+    package_base_dir: packages/foreman/
     releasers:
-    - koji-katello
+    - koji-foreman
+katello_client_packages:
```

Comment:
Possible to have the formatter put empty lines between groups? Would be easier to spot start and stop of a group that way.

Reply:
agreed